.changes/next-release/feature-AWSSDKforJavav2-65d4de2.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "AWS SDK for Java v2",
```

Comment:
nit: `Apache HTTP Client 5`?

Reply:
I was thinking since `Apache HTTP Client 5` did not exist yet and we are introducing a new feature in "AWS SDK for Java v2" I added category as "AWS SDK for Java v2"
Will change this to `Apache HTTP Client 5`